tests/rpc/rpc_05-state_test.go

```diff
@@ -34,10 +36,17 @@ func TestStateRPCResponseValidation(t *testing.T) { //nolint:tparallel
 	t.Run("state_call", func(t *testing.T) {
 		t.Parallel()
 
-		const params = `["", "","0x580d77a9136035a0bc3c3cd86286172f7f81291164c5914266073a30466fba21"]`
-		var response modules.StateCallResponse
+		const params = `["Core_version", "0x"]`
```

Comment:
Out of curiosity, why is 0x the case we are testing with?

Reply:
`Core_version` doesn't take any parameters